aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -399,4 +437,8 @@ public boolean isS3ChecksumEnabled() {
   public void setS3ChecksumEnabled(boolean eTagCheckEnabled) {
     this.isS3ChecksumEnabled = eTagCheckEnabled;
   }
+
+  public boolean isLakeFormationEnabled() {
```

Comment:
This should be naemd `glueLakeFormationEnabled()`, also need a `setGlueLakeFormationEnabled` method for setting the value, mostly for testing purpose.